src/bit-fields/Cargo.toml

```
@@ -0,0 +1,39 @@
+[package]
+name = "bit-fields"
+version = "0.1.0"
+authors = ["Amazon Firecracker team <firecracker-devel@amazon.com>"]
+edition = "2018"
+license = "Apache-2.0"
+keywords = ["bit", "bitmask", "bitflags", "flags"]
+repository = "https://github.com/firecracker-microvm/firecracker"
+homepage = "https://github.com/firecracker-microvm/firecracker"
+documentation = "https://docs.rs/bit-fields"
+description = "A macro to generate structures which support bit flags and sub-bytes ranges."
+exclude = ["/src/main.rs"]
```

This should probably be `src/example.rs`? (there is no `main.rs`)

```suggestion
exclude = ["/src/example.rs"]
```